activerecord/test/cases/dirty_test.rb

```diff
@@ -941,6 +941,51 @@ def check_around
     assert_not_predicate person, :changed?
   end
 
+  test "partial insert off with unchanged default function attribute" do
+    partial_inserts_was = Aircraft.partial_inserts
+    Aircraft.partial_inserts = false
+    Aircraft.connection.add_column(:aircraft, :manufactured_at, :timestamp, default: -> { "now()" })
```

Comment:
I think it would be cleaner to just change the model.

Reply:
moved the column to `aircraft` schema.